topo_processor/cli/geostore/add.py

```diff
@@ -42,15 +50,21 @@
     is_flag=True,
     help="Use verbose to display debug logs",
 )
-def main(source: str, role: str, commit: bool, verbose: bool) -> None:
+def main(source: str, datatype: str, role: str, commit: bool, verbose: bool) -> None:
     """Create or add a new version of an existing dataset to the Geostore for the source (survey) passed as argument."""
     start_time = time_in_ms()
+    data_type = DataType(datatype)
     logger = get_log()
     logger.info("geostore_add_started", source=source)
 
     if not verbose:
         set_level(LogLevel.info)
 
+    if data_type == DataType.IMAGERY_HISTORIC:
```

Comment:
That's a shame that there is no information about the data type in the STAC metadata. Or I missed it?
It would be more convenient to get this information from the `collection.json` so the user does not have to provide it in the command.
Without having this possibility, the approach you took would be the on I'd take too...

Reply:
We could infer it from the collection.json e.g. the `stac_extensions` :thinking:

Reply:
Yes I thought about that too but that looks too "guessing"